gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```

Comment:
@blackdrag @sclassen as this was just added, why is not the most recent gradle version used here? :)

Reply:
because that version is not compatible with the used plugin for NPM